src/main/java/com/pingcap/tikv/expression/scalar/Mod.java

```diff
@@ -40,4 +40,9 @@ public DataType getType() {
     // TODO: Add type inference
     throw new UnsupportedOperationException();
   }
+
+  @Override
+  ScalarFuncSig getSignature() {
+    throw new UnsupportedOperationException();
```

Comment:
Same here.

Reply:
We don't need this for now. Class removed.